aspnetcore/tutorials/first-mvc-app/adding-model.md

```diff
@@ -411,6 +413,7 @@ Because the `Model` object is strongly typed (as an `IEnumerable<Movie>` object)
 
 ## Additional resources
 
+* [Use SQLite for development, SQL Server for production](?tabs=visual-studio-code#sqlite-dev)
```

Comment:
@scottaddie  will this in page link work. With the default tab, need to specify the VSC tab.

Reply:
See my suggestion about putting the querystring at the end.